tensorbay/sensor/sensor.py

```diff
@@ -124,6 +146,18 @@ def dumps(self) -> Dict[str, Any]:
         Returns:
             A dict containing the information of the sensor.
 
+        Examples:
+            Init from *loads()* method, check below to see more details
```

Comment:
```lang=python
>>> # sensor is the object initialized from self.loads() method.
```
The principle to decide whether using a comment or separate description outside the doctest block is whether the content is about the whole example or only a variable inside the doctest block.